tests/e2e/tests/admin/signup.spec.ts

```diff
@@ -96,8 +96,11 @@ test.describe('Sign Up', () => {
   test('a user should be able to signup when the strapi instance starts fresh', async ({
     page,
   }) => {
-    await page.getByRole('button', { name: "Let's start" }).click();
+    await Promise.all([
```

Comment:
Using Promise.all with waitForLoadState and a click action works better to give clicks the time necessary to load the next page